src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/SetAzureCognitiveServicesAccount.cs

```diff
@@ -74,6 +85,51 @@ public class SetAzureCognitiveServicesAccountCommand : CognitiveServicesAccountB
         [AllowEmptyCollection]
         public string CustomSubdomainName { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Generate and assign a new Cognitive Services Account Identity for this storage account for use with key management services like Azure KeyVault.")]
+        public SwitchParameter AssignIdentity { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = false)]
+        [ValidateNotNullOrEmpty]
+        public IdentityType? IdentityType { get; set; }
+
+        [Parameter(HelpMessage = "List of User Owned Storage Accounts.", Mandatory = false)]
+        [ValidateNotNull]
+        [AllowEmptyCollection]
+        public string[] StorageAccountIds { get; set; }
```

Comment:
PowerShell is strict on the single/plural format of param name -- only when the length of it's value must > 1 can it be plural, so please use `StorageAccountId` instead

Reply:
Made the change.